Assets/Scripts/Model/Timing/AccessCard.cs

```diff
@@ -51,7 +51,7 @@ async private Task Trash()
 
         async private Task Steal()
         {
-            if (card is cards.corp.CorporateSalesTeam)
+            if (card.Type.Stealable != Stealable.CANNOT_STEAL)
```

Comment:
PS. We'd have to call `card.Stealable` rather than `card.Type.Stealable` in order to respect card-specific overrides.